src/Sulu/Bundle/ContentBundle/Content/Types/InternalLinks.php

```diff
@@ -78,9 +77,11 @@ public function read(
         $languageCode,
         $segmentKey
     ) {
-        $data = $node->getPropertyValueWithDefault($property->getName(), array());
-
-        $this->setData($data, $property, $webspaceKey, $languageCode);
+        $data = array();
+        if ($node->hasProperty($property->getName())) {
+            $data = $node->getProperty($property->getName())->getString();
+        }
```

Comment:
with this we get only an array of strings an no node will be loaded! thats enough  for us! with `getPropertyValueWithDefault ` we cannot define which we want to get!